exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -216,6 +218,10 @@ public boolean isTypeInferenceEnabled() {
     return options.getOption(TYPE_INFERENCE);
   }
 
+  public long getInSubqueryThreshold() {
+    return options.getOption(IN_SUBQUERY_THRESHOLD.getOptionName()).num_val;
```

Comment:
Change the declaration to: `public static final LongValidator IN_SUBQUERY_THRESHOLD ...`

and here, `return options.getOption(IN_SUBQUERY_THRESHOLD;`